client/app/domain/dns/DNS.html

```diff
@@ -122,6 +122,7 @@
                         data-ng-if="ctrlDomainDns.allowModification && !ctrlDomainDns.editMode">
                 </button>
                 <button class="btn btn-block btn-default" type="button"
+                        data-ng-attr-title="{{ctrlDomainDns.domain.managedByOvh && ctrlDomainDns.isDnssecEnable && tr('domain_tab_DNS_lock_button_title') || undefined }}"
```

Comment:
I don't get this, wouldn't `ctrlDomainDns.domain.managedByOvh && ctrlDomainDns.isDnssecEnable ?tr('domain_tab_DNS_lock_button_title') : undefined` be more appropriate ?